python/ray/_private/memory_monitor.py

```diff
@@ -147,14 +147,18 @@ def get_memory_usage(self):
             assert used_gb >= 0
         return used_gb, total_gb
 
+    def disabled(self) -> bool:
+        return (
+            ray._config.memory_monitor_interval_ms() > 0
+            or "RAY_DEBUG_DISABLE_MEMORY_MONITOR" in os.environ
```

Comment:
hmm weird we have 2 env vars... do you know why @ericl ?

Reply:
Probably one was for legacy reasons.